staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder.go

```diff
@@ -360,9 +354,9 @@ func (b *builder) buildRoute(root, path, httpMethod, actionVerb, operationVerb s
 		endpoints.AddObjectParams(b.ws, route, &metav1.ListOptions{})
 	case "put", "patch":
 		// TODO: PatchOption added in feature branch but not in master yet
```

Comment:
totally unrelated to this PR, and should be done in a different PR, but I just noticed this TODO... we're using PatchOptions now, so I think the put and patch cases should split and the patch case should use PatchOptions now

Reply:
opened https://github.com/kubernetes/kubernetes/pull/115339 stacked on top of this one to avoid conflicts from removing `disabledParams`